packages/dcos-diagnostics/extra/dcos-diagnostics-master.service

```diff
@@ -4,12 +4,16 @@ Description=DC/OS Diagnostics Master: aggregates and exposes component health
 EnvironmentFile=/opt/mesosphere/environment
```

Comment:
Why are any changes to this file necessary now?

Reply:
We discussed it. New API call agents endpoints and can't use it's hostname due to certifcate is issued only on IP, localhost, 127.0.0.1. Mesos listens only on IP so we need to inject IP obtained with `detect_ip` sript as a `--hostname`

Reply:
https://github.com/dcos/dcos/pull/6032/commits/727cd76eaae050c87ee960d1adce7377da4f4b78